src/Catalog/Monitoring/DummyResourceProvider.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Protocol.Core.Types;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring.Providers
+{
+    public static class DummyResourceUtils
```

Comment:
the word "dummy" can mean a lot of things

---
In reply to: [109520826](https://github.com/NuGet/NuGet.Services.Metadata/pull/151#discussion_r109520826) [](ancestors = 109520826)